react/Autosuggest/Autosuggest.js

```diff
@@ -122,10 +124,10 @@ export default class Autosuggest extends Component {
     };
 
     const allInputProps = {
+      ...omit(this.props, [ 'inputProps', 'labelProps', 'autosuggestProps' ]),
       ref: this.storeTextFieldReference,
       inputProps: enrichedInputProps,
-      labelProps: enrichedlabelProps,
-      ...omit(this.props, [ 'inputProps', 'autosuggestProps' ])
```

Comment:
Does it make any difference?  Are you expecting to overwrite anything with the results of the omit (or vice versa?)